From: Mike Hommey <mh@glandium.org>
Date: Fri, 31 Aug 2012 09:01:08 +0200
Subject: Allow webGL with mesa, assuming users will have updated to 8.0.4-2
 on wheezy

The version in squeeze-backports is not affected by CVE-2012-2864, and the
version in squeeze is blacklisted.
---
 widget/src/xpwidgets/GfxInfoX11.cpp |    4 ----
 1 file changed, 4 deletions(-)

diff --git a/widget/src/xpwidgets/GfxInfoX11.cpp b/widget/src/xpwidgets/GfxInfoX11.cpp
index a2d5e0b..27a0fd0 100644
--- a/widget/src/xpwidgets/GfxInfoX11.cpp
+++ b/widget/src/xpwidgets/GfxInfoX11.cpp
@@ -310,10 +310,6 @@ GfxInfo::GetFeatureStatusImpl(PRInt32 aFeature,
     }
 
     if (mIsMesa) {
-        if (aFeature == nsIGfxInfo::FEATURE_WEBGL_OPENGL) {
-            *aStatus = nsIGfxInfo::FEATURE_BLOCKED_DRIVER_VERSION;
-            aSuggestedDriverVersion.AssignLiteral("Not Mesa");
-        }
         if (mIsNouveau && version(mMajorVersion, mMinorVersion) < version(8,0)) {
             *aStatus = nsIGfxInfo::FEATURE_BLOCKED_DRIVER_VERSION;
             aSuggestedDriverVersion.AssignLiteral("Mesa 8.0");
